dev/NavigationView/NavigationView.cpp

```diff
@@ -3130,6 +3130,7 @@ void NavigationView::UpdateListViewItemSource()
     if (!dataSource)
     {
         dataSource = MenuItems();
+        UpdateSelectionForMenuItems();
     }
```

Comment:
Is the problem that we manage SelectedItem ourselves and ignore the one that ListView has? Can we instead try to listen for property changes on ListView.SelectedItem and sync those 2 up?

Reply:
I tried but it doesn't work. NavigationView listened to ListView.SelectionChanged after OnApplyTemplate, and the breakpoint was never been hit.
We have two way binding on NavView.SelectedItem and ListView.SelectedItem, and I didn't see NavView.SelectedItem property changed too during initialization.
I didn't dig into it since we have simple solution here.

The two way binding:
                                        <local:NavigationViewList
                                            ...
                                            SelectedItem="{TemplateBinding SelectedItem}"

Reply:
Yeah this seems fine but I keep seeing this type of problem reported across all Selector-based controls and I'm worried we aren't fixing the root cause. Something to keep in mind when you are fixing the next such bug in another control. :)

Reply:
Possible TreeViewItem has the same problem.